optimize/example_run.py

```diff
@@ -1,27 +1,63 @@
 #!/usr/bin/env python3
 
-from ParamFitter import ParamFitter
 import argparse
+import sys
+import traceback
+from torch.utils.data import DataLoader
 
-def main(param_list, input_file, lr, epochs, seed):
-    param_fit = ParamFitter(param_list, lr=lr) 
-    param_fit.make_target_sim(seed=seed)
-    param_fit.load_data(filename=input_file)
-    param_fit.fit(epochs=epochs)
+from .fit_params import ParamFitter
+from .dataio import TracksDataset
```

Comment:
I got a `ImportError: attempted relative import with no known parent package` here -- maybe we should make an `optimize/optimize` directory and move everything (except `example_run`) there?

Reply:
You should be able to run it as a python module using `-m` from the parent directory. Something like
```
python -u -m optimize.example_run --params lifetime --input_file data/edepsim-output.h5 --batch_sz 2
```

Reply:
Ah ok ok, that's fine then